cmd/cluster-agent/subcommands/start/command.go

```diff
@@ -297,46 +297,25 @@ func start(log log.Component,
 		return err
 	}
 
-	// Create event recorder
```

Comment:
This is a weird artifact, but mostly the RC config startup was moved before this.